package-lock.json

```diff
@@ -1912,7 +1912,8 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
Do we really need this change of `package-lock.json` in our repo? o.O

Reply:
Yes, as long as it will get changed after each install, I will keep committing its changes.